### PR TITLE
Add resources for admin tools containers and creating network policies

### PR DESCRIPTION
**Pull request by Demcheck**
## What was changed

This PR add option to create NetworkPolicies and add recources for admintools containers.

## Why?

In our company we use network policies in k8s clusters how must have option for all services.
Also within out clusters we have Kyverno that checking recources for all containers.

1. Closes 

2. How was this tested:

It has tested in our k8s cluster.

3. Any docs updates needed?

I think nothing need to update.

### CODE REVIEW

**Comment by ngnix** [comment]
Hey guys, do you have any update?

File: templates/network-policies.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        - podSelector:
+            matchLabels:
+              app.kubernetes.io/name: {{ include "temporal.name" . }}
+              app.kubernetes.io/instance: {{ .Release.Name }}
+      ports:
+        {{- range $service := (list "frontend" "history" "matching" "worker") }}
+        {{- $serviceValues := index $.Values.server $service }}
+        - protocol: TCP
+          port: {{ $serviceValues.service.port }}
+        {{- end }}
```
Comment (underrun) [inline templates/network-policies.yaml:30]: does this work if fontend, history, matching and worker all share the same port?

**Comment by robholland** [comment]
I'm happy to work with you on getting this merged, but please rebase against current master branch.

**Comment by robholland** [comment]
Please re-open if you decide to continue this work and rebase.

**Pull request closed without merging**